csunplugged/tests/resources/views/test_binary_cards_small.py

```diff
@@ -45,15 +45,24 @@ def test_binary_cards_small_resource_generation_valid_configurations(self):
         combinations = [dict(zip(valid_option_keys, product)) for product in itertools.product(*(valid_options[valid_option_key] for valid_option_key in valid_option_keys))]  # noqa: E501
         print()
         for combination in combinations:
+            print(combination)
             print("   - Testing combination: {} ... ".format(combination), end="")
-            url = base_url + query_string(combination)
+            url_combination = {}
+            for parameter in combination:
```

Comment:
Optional: It might be better to replace this code with the `util` function now to save this duplication.